docs/usage.md

```diff
@@ -425,6 +425,50 @@ field_name_mapping = {
 ```
 
 
+### Working with polymorphic resources
+
```

Comment:
This jumps straight into *how* to use polymorphic resources. While that is appropriate for a Usage page, if this is the only place in the documentation that describes polymorphic resources, I think an introductory paragraph providing context and *why* a user would need this would be beneficial. Also, are there going to be related packages that are needed for this work? If so, can we link to them? I'm thinking about `django-polymorphic` here.